apps/vr-tests-react-components/src/stories/Spinner.stories.tsx

```diff
@@ -12,7 +12,7 @@ const InvertedWrapper: React.FC = ({ children }) => {
 };
 
 storiesOf('Spinner converged', module)
-  .addDecorator(TestWrapperDecoratorNoAnimation)
+  .addDecorator(TestWrapperDecoratorPauseAnimation)
```

Comment:
Combine TestWrapperDecorator and PauseAnimationDecorator together. You'll also need to add a StoryWright node to crop the screenshots.
```suggestion
  .addDecorator(TestWrapperDecorator)
  .addDecorator(PauseAnimationDecorator)
  .addDecorator(story => (
    <StoryWright steps={new Steps().snapshot('default', { cropTo: '.testWrapper' }).end()}>{story()}</StoryWright>
  ))
```